src/logic/safe/utils/shouldSafeStoreBeUpdated.ts

```diff
@@ -0,0 +1,108 @@
+import { SafeRecordProps } from 'src/logic/safe/store/models/safe'
+import { sameString } from 'src/utils/strings'
+import isEqual from 'lodash.isequal'
+
+export const shouldSafeStoreBeUpdated = (
+  newSafeProps: Partial<SafeRecordProps>,
+  oldSafeProps?: SafeRecordProps,
+): boolean => {
+  if (!oldSafeProps) return true
+
```

Comment:
have you tried running the tests with `return isEqual(newSafeProps, oldSafeProps)` ?

Reply:
tests will fail in that case :(